src/components/MultiHopTrade/components/MultiHopTradeConfirm/MultiHopTradeConfirm.tsx

```diff
@@ -128,7 +133,15 @@ export const MultiHopTradeConfirm = memo(() => {
           </WithBackButton>
         </CardHeader>
         {isTradeComplete ? (
-          <TradeSuccess handleBack={handleBack}>
+          <TradeSuccess
+            handleBack={handleBack}
+            titleTranslation={
+              isArbitrumBridgeWithdraw ? 'bridge.arbitrum.success.tradeSuccess' : undefined
+            }
+            descriptionTranslation={
+              isArbitrumBridgeWithdraw ? 'bridge.arbitrum.success.tradeComplete' : undefined
+            }
```

Comment:
I wanted to put this inside some memos but it makes no points for now as this is the only bridge we need this system for at this point

Reply:
`titleTranslation` and `descriptionTranslation` is misleading here, this is a translation key without interpolation args. See https://github.com/shapeshift/web/pull/7992/files#r1813308736

Reply:
Done in [3d19513](https://github.com/shapeshift/web/pull/7992/commits/3d1951386bf5e0dba41a1d9082f2a0598502f0ae) I guess if I understood it right